src/vault.py

```diff
@@ -438,6 +438,25 @@ def ddos(args):
 
     args = parser.parse_args()
 
+    if args.all:
+        if args.url:
+            url = check_url(args.url)
+            ssl(args)
+            info(args)
+            fuzz(args)
+            comment(args)
+            xss(args)
+            lfi(args)
+
+        if args.ip:
+            ip = check_ip(args.ip)
```

Comment:
Same here.
Instead a better solution would be to add `args.all` after `args.url` and `args.ip` have been validated. (After Line 445 (master branch))